docs/openapi/lpa-codes-openapi-v1.yml

```diff
@@ -290,34 +291,25 @@ components:
   schemas:
     CodeCreated:
       properties:
-        code:
-          type: string
-        status:
-          type: string
-        id:
-          type: string
-      required:
-        - code
-    CodeRevoked:
-      properties:
-        code:
-          type: string
-        status:
-          type: string
-        id:
-          type: string
+        codes:
+          type: array
+          items:
+            type: object
+            properties:
+              lpa:
+                type: string
+              actor:
+                type: string
+              code:
+                type: string
```

Comment:
Allowing multiple rows to be sent/retrieved at once complicates the error handling a bit. If one of the codes fails to generate (whether that's from bad input or a system error), would the successful codes be returned? Or just a general 4xx with no codes?

Reply:
The whole thing would fail with a 5XX I suppose. I don't think a single code could fail to generate really, unless the generation algorithm gets more complex. 
Spoiler - it's just picking 12 random items from a list of allowed chars at the moment..!

Reply:
All-or-nothing would probably be easier for us to handle. Then I suppose we'd just try again or manually send out the letters.

The situation I considered for a single code failing was if one bit of data was invalid (e.g. an impossible date of birth), but I guess that might get kicked out immediately by validation.

Reply:
It would be kicked out by validation but I'm not sure I understand the reason for needing to create multiple codes at a time. Can you clarify what the reason behind this would be Jen? I think I missed the requirement.

Reply:
Yeh non-dates would not even make it through, you'd get a 400 before it even started generating codes.